test/neo/main.d

```diff
@@ -113,6 +114,50 @@ class Test : Task
         enforce(ok, "Get request failed");
         enforce(value == cast(void[])"hello");
     }
+
+    /***************************************************************************
+
+        Runs a simple test where three records are written to the node with Put
+        and then fetched with GetAll.
+
+    ***************************************************************************/
+
+    private void testPutGetAll ( )
+    {
+        mstring msg_buf;
+
+        // Add some records.
+        cstring[hash_t] records =
+            [0x1 : "you"[], 0x2 : "say", 23 : "hello"];
+        foreach ( key, value; records )
+        {
+            auto ok = this.client.blocking.put(key, value,
+                ( Client.Neo.Put.Notification info, Client.Neo.Put.Args args )
+                {
+                    formatNotification(info, msg_buf);
+                    Stdout.formatln("Put: {}", msg_buf);
```

Comment:
It is the question if the printing out in tests is worth it. I feel `enforce` is enough, since this is just a noise.